### PR TITLE
Tweak Find in File Progress hits text position

### LINKED ISSUE #13426: French translation text is cropped in search window
### Steps to Reproduce the Issue

1. Open a text file
2. Hit Ctrl+F
3.  The following text is cropped: "Dans la séléctio" (see image)

image

### Debug Information
Notepad++ v8.5.1   (64-bit)
Build time : Mar 21 2023 - 19:16:10
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Users\xxx\Desktop\test.CSV"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Education (64-bit)
OS Version : 22H2
OS Build : 19045.2728
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)

### LINKED ISSUE #15244: [BUG]  auto-completion,  "html/xml close tag" is cut for some languages
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

auto-completion,  "html/xml close tag" is cut for some languages (French, Italian, Portuguese,...)

immagine

immagine

### Steps To Reproduce

1.  enter setting, preferences in French, Italian, ecc
2.  label cut
3.  

### Current Behavior

label cut

### Expected Behavior

give more space

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Notepad++\change.log" 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Home (64-bit)
OS Version : 23H2
OS Build : 22631.3672
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ozone10**
- increase width for Find dialog In selection checkbox
- increase width for Preference Autocompletion html/xml close tag checkbox

fix #15244
fix #13426
ref https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15187#issuecomment-2159602146

### CODE REVIEW

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc [hunk 1]
```diff
@@ -57,8 +57,8 @@ BEGIN
     CONTROL         "Re&gular expression",IDREGEXP,"Button",BS_AUTORADIOBUTTON,12,167,78,10
     CONTROL         "&. matches newline",IDREDOTMATCHNL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,93,167,101,10
 
-    GROUPBOX        "",IDC_REPLACEINSELECTION,192,50,180,23
-    CONTROL         "In select&ion",IDC_IN_SELECTION_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,200,58,59,10
+    GROUPBOX        "",IDC_REPLACEINSELECTION,162,50,180,23
+    CONTROL         "In select&ion",IDC_IN_SELECTION_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,170,58,90,10
```
Comment (donho) [inline PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc:61]: It makes "In selection" checkbox float in the middle of Mark section...
However, "In selection" checkbox in Mark section should belong to "Mark all" & "Clear all marks" button.
Have you any idea for the solution?

**Comment by donho** [comment]
A 5 days’ poll is created here:
https://community.notepad-plus-plus.org/topic/25874/poll-find-in-files-hits-position

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc (hunk 1 above)
Comment (ozone10) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc:61]: @donho 
maybe this
Screenshot 2024-06-11 180703

**Comment by donho** [comment]
It seems people prefer the version of this PR. Solution accepted.

**Pull request closed without merging**